components-api/src/main/scala/pl/touk/nussknacker/engine/ModelConfig.scala

```diff
@@ -46,7 +46,7 @@ object ModelConfig {
   private def parseLiveDataPreviewMode(config: Config): LiveDataPreviewMode = {
     if (config.getOrElse("liveDataPreview.enabled", false)) {
       LiveDataPreviewMode.Enabled(
-        maxNumberOfSamples = config.getOrElse("liveDataPreview.maxNumberOfSamples", 10),
+        maxNumberOfRecords = config.getOrElse("liveDataPreview.maxNumberOfSamples", 10),
```

Comment:
Shouldn't the config entry `liveDataPreview.maxNumberOfSamples` also be renamed?